src/pages/BeersList.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+
+function BeersList() {
+  const [beers, setBeers] = useState([]);
+
+  const getBeers = async () => {
+    try {
+      let response = await axios.get(
+        "https://ih-beers-api2.herokuapp.com/beers"
+      );
+      console.log(response.data);
+      setBeers(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    getBeers();
+  }, []);
+
+  return (
+    <div>
+      {beers.map((beer) => {
```

Comment:
Here just be careful when rendering something without condition. This works but if for some reason the call takes more time it will break the code